liam/files/css.css

```diff
@@ -95,6 +107,10 @@ div {
     margin-right: auto;
 }
 
+#doodle {
+    min-width: 900px;
```

Comment:
![image](https://user-images.githubusercontent.com/5946409/39176983-71add65c-47ae-11e8-9920-347cf5879e88.png)
This is what happens on mobile

Reply:
That's your code translated from JS to CSS.